database/mysql.go

```diff
@@ -42,20 +45,33 @@ func (e *Mysql) Setup() {
 	}
 
 	// 是否开启详细日志记录
-	orm.Eloquent.LogMode(viper.GetBool("settings.gorm.logMode"))
+	// orm.Eloquent.LogMode(viper.GetBool("settings.gorm.logMode"))
```

Comment:
This specific line is the only where we need an input from the customer. Up to version 1, it should have been set if to log or not. From version 2, there are levels of logging, and so if the logMode is set, there is a need to decide on the level of it (Info, Error, Warn).